gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -325,6 +343,18 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  lymph_node_dissection_method:
+    enum:
+      - Functional (Limited) Neck Dissection
+      - Modified Radical Neck Dissection
+      - Radical Neck Dissection
+
+  lymph_node_dissection_site:
+    enum:
+      - Neck, Left
+      - Neck,Right
```

Comment:
do we need an extra space between the comma and "R"? (jira is the same)

Reply:
copying error, thanks for point out. Fixed.